docroot/sites/all/modules/features/forms/forms.info

```diff
@@ -858,7 +858,6 @@ features[variable][] = field_bundle_settings_field_collection_item__field_employ
 features[variable][] = field_bundle_settings_field_collection_item__field_employment_history_uli
 features[variable][] = field_bundle_settings_field_collection_item__field_h_credit_card_payments
 features[variable][] = field_bundle_settings_field_collection_item__field_remote_site_details
-features[views_view][] = submissions_alumni_postings
```

Comment:
This is the kind of thing you should be looking for in your commits. I've added back this line before merging. Please keep a closer eye on what your features update changes are before committing.